### PR TITLE
balances the netherworld link, spawners can now spawn from a pool of mobs, and the netherworld link can now summon all three types of netherworld critters. (and mining spawner cleanup for admin events i guess)

### PR DESCRIPTION
**Pull request by ghost**
Someone added a netherworld link to a map, which means I have the balance it now. It was originally intended as a wizard weapon, summoning them but I didn't know wizard code and flooding deadly mobs can be done way better with the cult shards (and it looks much cooler so lets just stick with that).

So my next idea was to remove it but people really liked the link so I kept it as an admin spawn since all the mi-gos spam noise and i liked that a lot.

But the problem is I just left the values as admin spawn and now that it's on a map 30 mi-gos with 2 of them spawning every 5 seconds doesn't sound fun or fair. would one portal instead of 2 fix that? no, it really is that strong that i'd consider a minute for a 3-hit-crit monster fair since it can spawn so many mi-gos

DURR BUT IS IT LORE FRIENDLY??
yes. yes it is. netherworld creatures are vastly weaker than they are in the nether because travelling to this realm expends much of their strength. if anything, having them pour through like an orc horde wasn't really that realistic.

the mining cleanup seems stupid something i don't need to touch but it actually made no sense since the desc said it spawned all types of monsters but it was actually the goldgrub den.

### CODE REVIEW

**Comment by Armhulen** [comment]
Also, I fixed a tiny thing with the gibspawner. it spammed human faces over and over as it's list of gibs and that was really weird. this new one doesn't do that

**Comment by Armhulen** [comment]
I also might make it pop out all three kinds of netherworld mobs, just requires me to rewrite some spawner code

**Comment by Buggy123** [comment]
Wait so it now produces 1 mob a minute instead of 1 every 5 seconds? That seems like a bit too much of a nerf.

**Comment by Armhulen** [comment]
i could make it 30 seconds but the netherworld link has an unusually high amount of creatures allowed to roam around so i think it's fair.

**Comment by Spaceinaba** [comment]
ignore buggy he doesnt understand

**Comment by Buggy123** [comment]
Rude

**Comment by Armhulen** [comment]
@Buggy123 i didn't thumbby down ur pr

**Comment by Buggy123** [comment]
Oh okay, thumbsup it is

**Comment by MMMiracles** [comment]
if the map you're talking about is snowdin, balance probably isn't an issue considering thats an away-mission.

**Comment by Armhulen** [comment]
yeah but i'd prefer if it was in a more balanced state anyway for admin stuff. besides why NOT be more fair to people exploring, it's my fluff anyways

**Comment by MMMiracles** [comment]
i actually dropped the max amount of mobs per spawner in snowdin, but this works too

File: code/modules/mob/living/simple_animal/spawner.dm [hunk 1]
```diff
@@ -8,7 +8,8 @@
 	var/max_mobs = 5
 	var/spawn_delay = 0
 	var/spawn_time = 300 //30 seconds default
-	var/mob_type = /mob/living/simple_animal/hostile/carp
+	var/mob_types = list(/mob/living/simple_animal/hostile/carp)
+	var/chosen_mob_type
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/spawner.dm]: Why is this on the object and not localized within the proc?

Comment (Armhulen) [reply code/modules/mob/living/simple_animal/spawner.dm]: yeah i can move it over sure

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

**Comment by vuonojenmustaturska** [comment]
It's a webeditor PR, but the changes seem simple enough

**Pull request merged**